shared/ui/core/src/test/java/com/ivy/ui/FormatMoneyUseCaseTest.kt

```diff
@@ -23,33 +23,80 @@ class FormatMoneyUseCaseTest {
     enum class MoneyFormatterTestCase(
         val amount: Double,
         val showDecimal: Boolean,
+        val shortenAmount: Boolean,
         val locale: Locale,
         val expectedOutput: String
     ) {
         ENG_SHOW_DECIMAL(
             amount = 1000.12,
             showDecimal = true,
+            shortenAmount = false,
             locale = Locale.ENGLISH,
             expectedOutput = "1,000.12"
         ),
         ENG_HIDE_DECIMAL(
             amount = 1000.12,
             showDecimal = false,
+            shortenAmount = false,
             locale = Locale.ENGLISH,
             expectedOutput = "1,000"
         ),
         GERMAN_SHOW_DECIMAL(
             amount = 1000.12,
             showDecimal = true,
+            shortenAmount = false,
             locale = Locale.GERMAN,
             expectedOutput = "1.000,12"
         ),
         GERMAN_HIDE_DECIMAL(
             amount = 1000.12,
             showDecimal = false,
+            shortenAmount = false,
             locale = Locale.GERMAN,
             expectedOutput = "1.000"
         ),
+        ENGLISH_100K_SHORT_AMT(
+            amount = 130000.10,
+            showDecimal = true,
+            shortenAmount = true,
+            locale = Locale.ENGLISH,
+            expectedOutput = "1.30k"
+        ),
+        ENGLISH_MILLION_SHORT_AMT(
+            amount = 1230000.10,
```

Comment:
use `_` in the number otherwise the test isn't readable and do that for all test cases

Reply:
Done